recipe/meta.yaml

```
@@ -1,12 +1,11 @@
-{% set version = "1.80.0" %}
-{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
+{% set boost_cpp_version = boost_cpp_version|default("1.80.0") %}
```

Aligning the version here with the one used in [the ABI migration]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/0104f2bbe765cf7f1888ee7f4478a28fa95cdaef/recipe/conda_build_config.yaml#L276 ) since this will also produce 1.78.0 packages

```suggestion
{% set boost_cpp_version = boost_cpp_version|default("1.78.0") %}
```

That said, this value is only here to allow re-rendering to work. It gets replaced at build time with the correct version.